### PR TITLE
MM Ported Over LDP to MetaDB Instance subjects

### PR DESCRIPTION
**Pull request by jenmawe**
New ported over derived table for metadb to extract subjects from FOLIO instance records.

### CODE REVIEW

**Review by stdombek — APPROVED** [review]
- [X] Query runs
- [X] Data output makes sense
- [X] Data types look correct
- [X] Query logic seems correct
- [X] Column names make sense and are spelled correctly
- [X] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [X] query has indexes on columns
- [X] vacuum analyze has been added to the bottom
- [X] table appears in runlist.txt at an appropriate point

Good work! Please check the type casting. I put approved because it's just a small thing.

**Review by adoerrer — APPROVED** [review]
Great work, thanks Jen!

- [x] Pull request has a complete description
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary


Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

Good work! Please check the type casting. I put approved because it's just a small thing.

**Comment by stdombek** [comment]
All correct, ready for merging.

File: sql_metadb/derived_tables/runlist.txt [hunk 1]
```diff
@@ -23,6 +23,7 @@ instance_formats.sql
 instance_editions.sql
 instance_languages.sql
 instance_nature_content.sql
+instace_subjects.sql
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/runlist.txt:26]: Typo in file name.  I will make the change.

**Pull request merged**